CMakeLists.txt

```diff
@@ -749,3 +749,5 @@ if(CCACHE_FOUND)
         set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
         set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
 endif(CCACHE_FOUND)
+
+INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/misc/zoneminder.desktop DESTINATION share/applications)
```

Comment:
Instead of hard coding the destination share/applications, can we use ${CMAKE_INSTALL_DATAROOTDIR}/applications instead? That's how we've done it elsewhere.